src/RtspConnection.c

```diff
@@ -665,6 +665,9 @@ static int parseOpusConfigFromParamString(char* paramStr, int channelCount, POPU
     int i;
 
     // Set channel count (included in the prefix, so not parsed below)
+    if (channelCount > AUDIO_CONFIGURATION_MAX_CHANNEL_COUNT) {
+        channelCount = AUDIO_CONFIGURATION_MAX_CHANNEL_COUNT;
+    }
```

Comment:
Thanks for the fix. I took care of it slightly differently (failing rather than clamping to the max).